doc/graph/fusion_patterns/sdpa_with_compressed_kv.md

```diff
@@ -0,0 +1,118 @@
+SDPA with Compressed Key and Value {#dev_guide_graph_compressed_sdpa}
```

Comment:
nit: dev_guide_graph_sdpa_compressed_kv to better align with the subject and content.